### PR TITLE
CHESS-6 - Fen to array notation

### PR DESCRIPTION
**Pull request by burmazovic**
Made fenToArray function for taking a FEN notation value and converting it to an Array.

### CODE REVIEW

File: src/components/Field.js [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
             color: piece === piece.toLowerCase()? 'black':'white'
         }
-    }
+    };
+    // FEN to Array
+    fenToArray = fenValue => {
+        let fenArray = [];
+        fenValue.split('').forEach( element => {
+            if (Number.isInteger(parseInt(element))) {
+                for (let i = 0; i < parseInt(element); i++) {
```
Comment (ban17) [inline src/components/Field.js]: Good logic! Just array need to have same structure as in Table component.

File: src/helpers/FenToArray.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            for (let i = 0; i < parseInt(element); i++) {
+                fenArray.push('')
+            }
+        } else {
+            fenArray.push(element)
+        }
+    });
+    return fenArray
+        .filter(piece => piece !== '/')
+        .map( field => ({id:'', x:'', y:'', piece:field}));
```
Comment (ban17) [inline src/helpers/FenToArray.js]: This need to have real values, not empty.

File: src/App.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import Table from './components/Table/Table'
 class App extends React.Component {
   render () {
     return (
-        <Table />
+        <Table  />
```
Comment (ban17) [inline src/App.js]: Redundant change.

File: src/helpers/FenToArray.js (hunk 2 above)
Comment (burmazovic) [reply src/helpers/FenToArray.js]: Added.

**Pull request merged**